themes/vue/source/css/index.styl

```diff
@@ -62,7 +62,7 @@ body
     margin: 1em 0
     font-size: 1.05em
     font-weight: 600
-    letter-spacing: .1em
+    letter-spacing: 0.04em
```

Comment:
Why not reset to 0? We still have empty pixels: 

![image](https://user-images.githubusercontent.com/5158436/58747753-b93f6580-8484-11e9-8ead-08a485e91fd7.png)

Reply:
The page that was rendered to me did not have an empty pixel
But now letter-spacing remove